src/bin/nydusd/fusedev.rs

```diff
@@ -255,8 +256,7 @@ impl FusedevDaemon {
         let thread = thread::Builder::new()
             .name("fuse_server".to_string())
             .spawn(move || {
-                if let Err(err) = s.svc_loop(&inflight_op) {
-                    warn!("fuse server exits with err: {:?}, exiting daemon", err);
+                if let Err(_err) = s.svc_loop(&inflight_op) {
```

Comment:
Have any other errors need to be printed?

Reply:
I observed that the error message is mainly issued for quit. 
And in case real error happens, the caller should print the error message.